gke-tpu-examples/single-host-inference/jax/stable-diffusion/app.py

```diff
@@ -0,0 +1,42 @@
+# Use flask to download a file from the serving model.
```

Comment:
can we add a readme file explaining how to deploy the server and start the app? for the server, it would suffice to point to GKE tpu provisioning pages and simply show the kubectl command.

Reply:
Added a README with links to the `TPU in GKE` doc. However, I didn't add specific instructions since the code here is for a quickstart doc, which has step-by-step instructions for deploying and starting the model/app. Once the quickstart doc is published, we can add a link in the README as well.